<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>399</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>435</width>
     <height>399</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>435</width>
     <height>399</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>461</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 11, 75);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>RESUMEN RONDA NÂ°</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_post_ronda">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>CLIENTES PERDIDOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>CLIENTES ATENDIDOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>DINERO ACUMULADO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_post_perdidos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(58, 0, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 40, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_post_dinero">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(97, 97, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>REPUTACION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>5/5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_post_salir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_post_guardar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_post_continuar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONTINUAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_post_salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>salir_juego()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_post_guardar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>guardar_partida()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_post_continuar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>continuar_partida()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>salir_juego()</slot>
  <slot>guardar_partida()</slot>
  <slot>continuar_partida()</slot>
 </slots>
</ui>
